tests/e2e/subnet-evm-genesis.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
can we improve this and download it in some way from subnet-evm ? is this subnet-evm version dependant? what do you think @dasconnor  ?

Reply:
I think it's way better to maintain test files in the repo. If needed, we update it here.

Reply:
Genesis files should definitely be fixed in this repo